aya/src/bpf.rs

```diff
@@ -314,6 +316,24 @@ impl<'a> BpfLoader<'a> {
         self
     }
 
+    /// Sets BPF verifier log level.
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// use aya::BpfLoader;
+    ///
+    /// let bpf = BpfLoader::new()
+    ///     .verifier_log_level(3)
+    ///     .load_file("file.o")?;
+    /// # Ok::<(), aya::BpfError>(())
+    /// ```
+    ///
+    pub fn verifier_log_level(&mut self, level: u32) -> &mut BpfLoader<'a> {
```

Comment:
I think instead of `u32` we should probably have an enum:

```
#[non_exhaustive]
#[repr(u32)]
pub enum VerifierLogLevel {
    Default = 4,
    Verbose = 1,
    Debug = 7,
}
```

See https://github.com/torvalds/linux/blob/f16214c102f0f64b2f3546e989498525bd7b7708/include/linux/bpf_verifier.h#L419. The Default/Verbose value correspond to what libbpf does too.

Reply:
Updated it [here](https://github.com/aya-rs/aya/pull/371/commits/edd80397dce46f6e2a4cc96bd951562987721e55), changed the names and added some valid configurations.

Let me know if you still prefer the above variants, I just felt like this it was slightly clearer.

Reply:
Could also change it up so that is similar to a bitmap, e.g. set it up like:

```rs
VerifierLogLevel::default().level_1(true).stats(true)
```

But might be too verbose for something like this.

Reply:
> Updated it [here](https://github.com/aya-rs/aya/pull/371/commits/edd80397dce46f6e2a4cc96bd951562987721e55), changed the names and added some valid configurations.
> 
> Let me know if you still prefer the above variants, I just felt like this it was slightly clearer.

I don't think this is cleaner - in fact it took me a good 10 minutes yesterday to understand why those names were used in the kernel and how they work. The names are bad, and I don't think we should surface the fact that log level s are implemented as a bitfield in the kernel all the way up to the aya API.

Reply:
We should also have a `Disable = 0` variant in addition to the ones I initially suggested tho